### PR TITLE
Adding new locations for teleportation

### PR DESCRIPTION
**Pull request by Specifer**
Nothing specific. I always wanted more tow locations, that's why.

I have tested this on my local server.

### CODE REVIEW

File: pawn/Resources/Callbacks/OnPlayer/OnPlayerCommandText.pwn [hunk 1]
```diff
@@ -955,11 +955,14 @@ public OnPlayerCommandText(playerid, cmdtext[]) {
...
+        SendClientMessage(playerid, COLOR_ORANGE, "9: Los Santos Airport   10: LV Main Bank 11: LV Train Station");
+        SendClientMessage(playerid, COLOR_ORANGE, "12: LV Fight Club       13: Bayside Marina 14: San Fierro Hospital");
+        SendClientMessage(playerid, COLOR_ORANGE, "15: San Fierro Pier     16: CJ's Hood"); 
+        SendClientMessage(playerid, COLOR_ORANGE, "17: Los Santos Observatory    18: Los Santos Beach");
         if (Player(playerid)->isAdministrator())
-            SendClientMessage(playerid, COLOR_ORANGE, "12: LV Fight Club 13: Balloon (/t only)");
+            SendClientMessage(playerid, COLOR_ORANGE, "19: Los Santos Golf Club 20: Balloon (/t only)");
         else
-            SendClientMessage(playerid, COLOR_ORANGE, "12: LV Fight Club");
+            SendClientMessage(playerid, COLOR_ORANGE, "19: Los Santos Golf Club");
```
Comment (RussellLVP) [inline pawn/Resources/Callbacks/OnPlayer/OnPlayerCommandText.pwn]: I agree with adding 13, 15 and 18. Could we only add these three for now?

Comment (Specifer) [reply pawn/Resources/Callbacks/OnPlayer/OnPlayerCommandText.pwn]: Ah, So you need it to be changed in everything. okey-dokey.

Comment (Specifer) [reply pawn/Resources/Callbacks/OnPlayer/OnPlayerCommandText.pwn]: @RussellLVP, Maybe we will let this full locations for administrators and deduce the locations of what you mentioned to players?

**Comment by RussellLVP** [comment]
No, I don't think we should do that.

When introducing a feature like this, where locations are identified with a number, players grow something we call _muscle memory_ when wanting to do something. This is a really common psychological effect: our players don't think "hey, I want to go to the ship, let's enter `/taxi 0`." Instead, it's something they do intuitively when spawning whilst wanting to fight.

https://en.wikipedia.org/wiki/Muscle_memory

The consequence of that is that changing the behaviour of such numbers, like you do with Id 13 (previously the baloon, now Bayside Marina), is incredibly annoying to those who use them. This is forward looking as well: if we introduce these now, all of the [CP] gang will be trained to use `/taxi 18` all the time. This completely locks our ability in place to change anything about this.

If we want a more powerful `/taxi` feature, I would rather figure out if we can enable players to teleport to _anywhere_. Esc -> Map -> Right click on any position, and poof, you teleport. I don't know whether that's the right idea just yet, but growing the number of locations available to these commands will never let us reach a comprehensive result.

**Comment by Specifer** [comment]
Ok, I understand!

**Review by RussellLVP — APPROVED** [review]

**Comment by RussellLVP** [comment]
Thank you!

**Pull request merged**